examples/jsm/nodes/lighting/EnvironmentNode.js

```diff
@@ -44,7 +44,8 @@ class EnvironmentNode extends LightingNode {
 
 		//
 
-		const intensity = reference( 'envMapIntensity', 'float', builder.material ); // @TODO: Add materialEnvIntensity in MaterialNode
+		const envMap = builder.material.envMap;
+		const intensity = envMap ? reference( 'envMapIntensity', 'float', builder.material ) : reference( 'environmentIntensity', 'float', builder.scene ); // @TODO: Add materialEnvIntensity in MaterialNode
```

Comment:
`Scene.environmentIntensity` should only affect `Scene.environment`, not materials with a configured `envMap` property. I'm not sure the code in `EnvironmentNode` adheres to this policy.

Reply:
Yea, it seems the code in `EnvironmentNode` is indeed correct. I've just misread the line^^.

Reply:
The code in `EnvironmentNode` is indeed right. I've just misread the new line^^.